apache/incubator-zeppelin
# [ZEPPELIN-840] Scalding interpreter that works in hdfs mode

### What is this PR for?
Scalding interpreter that works in hdfs mode

### What type of PR is it?
Improvement

### Todos

### What is the Jira issue?
[ZEPPELIN-840](https://issues.apache.org/jira/browse/ZEPPELIN-840)

### How should this be tested?
1. The remote interpreter has to be run in on a system with Hadoop libraries.
1. Run "%scalding mode" and verify that it is Hdfs
1. Run command that create map-reduce job. For example, "TypedPipe.from(TextLine("/user/pwagle/testfile")).filter(x => x == "a").toList"

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes
